### ISSUE TITLE
Controller doesn't properly suspend/resume

### ISSUE THREAD

**Issue by Moosheplusplus** [powerbot/powerbot#1266]
When trying to suspend or resume a controller, it doesn't always successfully suspend or resume. It also won't properly update the toolbar, so if the controller is resumed, it will still offer the option to resume the script. Sometimes it will throw this error:

```
[SEVERE] java.lang.NullPointerException 
	org.powerbot.bot.AuX.long(yh:87)
	org.powerbot.bot.AuX.suspend(yh:351)
```

**Comment by Paris** [comment]
Fixed, same bug as #1279.

**Comment by Moosheplusplus** [comment]
@Paris I am still experiencing this.
